src/Behat/Gherkin/Loader/AbstractFileLoader.php

```diff
@@ -50,7 +50,7 @@ protected function findRelativePath($path)
      *
      * @param string $path Relative path
      *
-     * @return string
+     * @return false|string
```

Comment:
I couldn't see any better way to solve this - downstream users with PHPStan that extended this class will now get warnings, unless they fix it in a similar way to (for example), the `DirectoryLoader` changes in this PR.

Reply:
With tools like phpstan becoming more prevalent, it's an interesting question whether changing the phpdoc / annotation types of a method counts as a BC break or not...

In this case, however, it's clearly a bugfix - the previous value is obviously incorrect - so I think you're right to fix it like this.